packages/support/src/tools/_mixins.focus-text.scss

```diff
@@ -4,3 +4,10 @@
     color: $color-base;
     outline: 3px solid transparent
 }
+
+@mixin focus-text-inverse {
```

Comment:
Ah I just noticed this mixin file. Is the intention to allow others to use these mixins to achieve focus styles? Because if this is just to promote code reuse, I feel like the they are more appropriate as [sass placeholders](https://sass-lang.com/documentation/style-rules/placeholder-selectors)

Reply:
@bernardwang Oh cool. I haven't used placeholders much. I don't think you can use variables in placeholders, so that would be a blocker. I do like that the CSS doesn't output unless it's used.

The use case here is mostly for those times, like in the Help Drawer, when you need another element to take focus that normally wouldn't. In the Help Drawer, it's the heading after the link is clicked to open it.

Reply:
Ah I gotcha, if its intended to be used in other cases then I think a mixin is appropriate, I just wasnt aware of other use cases. 

Re: placeholders, I think they do allow sass variables, you just cant pass variables in for dynamic styles. Placeholder is also slightly more efficient with generated css, preventing duplication of CSS.